src/components/Article/Article.js

```diff
@@ -78,8 +84,10 @@ const Article = ({
         label,
       }}
       competenceGoals={renderCompetenceGoals(article, isTopicArticle, subject)}
+      competenceGoalTypes={Array.from(competenceGoalTypes)}
```

Comment:
Ikkje perfekt, men for å få distinkte verdier i lista gikk eg via et Set og så tilbake til Array.